public/routing/index.plugin.ts

```diff
@@ -22,3 +22,41 @@ const routingConfig: RoutingConfiguration = {
 };
 
 export default routingConfig;
+
+export const createPluginLinkBuilder: LinkBuilderFactory = (appKey?: string) => {
+  return (queryParams: string) => {
+    if (!appKey) {
+      throw new Error("App key is required for plugin links");
+    }
+
+    if (!AP) {
```

Comment:
I'm curious why this isn't `window.AP` and a linter doesn't freak out?

Reply:
I think `window` is the default scoping

Reply:
We've added it to the global definition in `routing/common.ts`